tests/system/providers/google/spanner/example_spanner.py

```diff
@@ -20,7 +20,7 @@
 Example Airflow DAG that creates, updates, queries and deletes a Cloud Spanner instance.
 
 This DAG relies on the following environment variables
-* GCP_PROJECT_ID - Google Cloud project for the Cloud Spanner instance.
+* PROJECT_ID - Google Cloud project for the Cloud Spanner instance.
 * GCP_SPANNER_INSTANCE_ID - Cloud Spanner instance ID.
 * GCP_SPANNER_DATABASE_ID - Cloud Spanner database ID.
 * GCP_SPANNER_CONFIG_NAME - The name of the instance's configuration. Values are of the
```

Comment:
We probably don't need this section about environment variables anymore - now the process is a lot more straightforward and if anything is required it will be listed in conftest.py . And some of those variables are not environment variables anymore ;)